<launch>

  <include file="$(find shared_control)/launch/gazebo.launch"/>
  <node pkg="shared_control" type="motion_manager.py" name="motion_manager"/>

  <include file="$(find shared_control)/launch/core.launch"/>

  <node pkg="shared_control" type="fake_bci.py" name="fake_bci" output="screen"/>

  <node pkg="rviz" type="rviz" name="rviz"
        args="-d $(find shared_control)/rviz/gazebo_test.rviz"/>

</launch>
